client/package.json

```diff
@@ -3,8 +3,12 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "axios": "^0.18.0",
+    "jest": "^23.6.0",
+    "prop-types": "^15.6.2",
```

Comment:
If jest was inadvertently added, please remove it while refactoring. Thanks David.